modmesh/pilot/_euler1d.py

```diff
@@ -732,7 +724,7 @@ def update_lines(self):
             self.entropy.update(adata=self.st.entropy_field,
                                 ndata=self.st.svr.entropy[_s])
         else:
-            for name, is_selected, *_ in self.plot_config.state:
+            for name, is_selected, *_ in self.plot_config._tbl_content:
                 if is_selected:
                     eval(f'(self.{name}.update(adata=self.st.{name}_field,'
```

Comment:
It will be good to get rid of `eval()` in Python because it is very hard to maintain.  But we can do it later.